backoff.py

```diff
@@ -321,6 +336,16 @@ def _handlers(hdlr, default=None):
     return defaults + [hdlr]
 
 
+# Evaluate arg that can be either a fixed value or a callable.
+def _maybe_call(f, *args, **kwargs):
```

Comment:
I was under the impression there wasn't a portable way to do it across 2.6 (we've got some 2.6 PRs so I'm trying to keep supporting this) 2.7 3.x but it looks like the callable built-in should work despite briefly going away in early 3.x.  I added a fixup to use it.